src/crystal/compiler_rt/mul.cr

```diff
@@ -1,5 +1,6 @@
 # :nodoc:
 private macro __mul_impl(name, type, n)
+  # Ported from llvm/compiler-rt:/lib/builtins/muloti4.c / mulodi4.c / mulosi4.c
```

Comment:
An exact link to the source code would be nice.

Reply:
Hmm.. I would have to change that accross the compiler-rt symbols
I had just taken the format that was previously used in mulodi3.cr.
But I agree, I will add exact sources soon.

Reply:
Fixed in #11206